tests/Tests/Models/ComplexHydration/Serie.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Models\ComplexHydration;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\Table;
+
+#[Table(name: 'serie')]
+#[Entity]
+class Serie
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
+    private int $id;
+
+    #[ORM\Column(type: 'string', length: 255)]
+    private string|null $libelle;
+
+    /**
+     * @var Collection<int, ItemSerie>
+     * @OneToMany(targetEntity="ItemSerie", mappedBy="serie")
```

Comment:
I'm asking again, why do some attributes also come with annotations?

Reply:
without I have PHPCS errors : 
26 | ERROR | [ ] Property \Doctrine\Tests\Models\ComplexHydration\Serie::$itemSeries does not have @var annotation for
    |       |     its value. (SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingAnyTypeHint)
 29 | ERROR | [ ] Property \Doctrine\Tests\Models\ComplexHydration\Serie::$serieImportators does not have @var
    |       |     annotation for its value. (SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingAnyTypeHint)

Reply:
oh just var is needed, I can supress OneToMany

Reply:
Yes, and there are more occurrences of more Doctrine annotations in there.